src/pages/users.vue

```
@@ -0,0 +1,156 @@
+});
+
+const handleSearch = async (query: string) => {
+    try {
+        const results = await searchUsers(query);
+        if (!results) {
+            users.value = []
+            return
+        }
+
+        users.value = results
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)